### ISSUE TITLE
osu!lazer crashes when switching a beatmap

### ISSUE THREAD

**Issue by iLiwe** [ppy/osu#4054]
[database.log](https://github.com/ppy/osu/files/2753223/database.log)
[network.log](https://github.com/ppy/osu/files/2753224/network.log)
[performance.log](https://github.com/ppy/osu/files/2753225/performance.log)
[runtime.log](https://github.com/ppy/osu/files/2753226/runtime.log)
Running Manjaro Deepin Edition 18.0.2
Intel Pentium N3540, Intel HD Graphics (Baytrail), 4GB RAM

I compiled osu!lazer via using the AUR pkgbuild, osu-lazer. When you click play then solo, it works great, loading everything. But when clicking on another beatmap, or just playing the current beatmap, the game crashes. Also gives a segfault error in terminal when it crashes 
[screenshot](https://user-images.githubusercontent.com/19897088/51087467-ddfca000-1764-11e9-8818-2e65d4cc5f41.png)
Hoping that the issue gets fixed easily.

**Comment by iLiwe** [comment]
Sure thing. I will report back if it works.

**Comment by iLiwe** [comment]
I compiled it myself using the command line, but it still doesn't work. Here are the logs again.
[database.log](https://github.com/ppy/osu/files/2753480/database.log)
[network.log](https://github.com/ppy/osu/files/2753481/network.log)
[performance.log](https://github.com/ppy/osu/files/2753482/performance.log)
[runtime.log](https://github.com/ppy/osu/files/2753483/runtime.log)

**Comment by iLiwe** [comment]
I have found the source of the issue. It is related with the "Disable GL driver error checking" option in driconf. I don't know how it is related, but if that options is true, then the game crashes as i explained. But if it's false, it doesn't crash. I kept that setting on for a really small performance boost, and it worked in every other game i played. It is really weird that this doesn't work in osu!lazer. Really sorry for opening an useless issue.
